src/components/NftTransfersTable.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <table class="nft-transfers-table">
+    <thead>
+      <tr>
+        <th>
+          Hash
+          <hint-tooltip>
+            {{ nftsHints.contractId }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Time
+          <hint-tooltip>
+            {{ nftsHints.transferTime }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Token ID
+          <hint-tooltip>
+            {{ nftsHints.transferTokenId }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Recipient
```

Comment:
I would add the Sender column before Recipient, I think it makes more sense logically

Reply:
Make sense, fixed